api/odl.py

```diff
@@ -837,12 +860,19 @@ def _detail_for_elementtree_entry(cls, parser, entry_tag, feed_url=None, do_get=
                 if rel == "self":
                     odl_status_link = link_tag.attrib.get("href")
                     break
+            if not odl_status_link:
+                # TODO: This is a hack necessary because Feedbooks
+                # doesn't provide the status link yet. Once that's fixed,
+                # this link will be present with rel="self", just like
+                # in the unit tests.
+                odl_status_link = "https://license.feedbooks.net/copy/status/?uuid=" + identifier.replace("urn:uuid:", "")
 
             if odl_status_link:
                 ignore, ignore, response = do_get(odl_status_link, headers={})
                 status = json.loads(response)
-                remaining_checkouts = status.get("checkouts", {}).get("left")
-                available_checkouts = status.get("checkouts", {}).get("available")
+                checkouts = status.get("checkouts", {})
+                remaining_checkouts = checkouts.get("left", 0)
```

Comment:
I figured defaulting these values to 0 was better than `None`.

Reply:
Hmm, I think if the book has unlimited checkouts, those values won't be in the status document and it's supposed to be `None`.

Reply:
Good to know, thanks.